postgres_ffi/src/pg_constants.rs

```diff
@@ -214,10 +214,9 @@ pub const PGDATA_SUBDIRS: [&'static str; 22] = [
     "pg_logical/mappings",
 ];
 
-pub const PGDATA_SPECIAL_FILES: [&'static str; 4] = [
+pub const PGDATA_SPECIAL_FILES: [&'static str; 3] = [
     "pg_hba.conf",
     "pg_ident.conf",
```

Comment:
Why this change?

Reply:
Sorry, reverted.
We need postgresql.conf file to be able to perform recovery: wal_acceptors addresses, timeline and tenant identifiers are specified here. I expected that this file is always generated by control panel/test scripts, so there is no need to include empty postgresq.conf file in tarball. But tests failures shows that this assumption was wrong. So I restoredincluding emoty postgresql.conf in basebackup and just extract titwith -k tar option (do not overwrite existed files.

This is how node is started in @ololobus test:
```
# Remove compute datadir, initialize it using basebackup and start
init_compute () {
    rm -rf zenith_tmp/compute
    mkdir zenith_tmp/compute

    echo "
listen_addresses='0.0.0.0'
password_encryption=md5
max_connections=100
max_wal_senders=10
max_replication_slots=10
hot_standby=on
wal_level=replica
wal_sender_timeout=0
wal_keep_size=256MB
shared_buffers=256MB
wal_log_hints=on
fsync=off
port=54325
shared_preload_libraries='zenith'
synchronous_standby_names='walproposer'
wal_acceptors='127.0.0.1:6501,127.0.0.1:6502,127.0.0.1:6503'
zenith.zenith_tenant='${TENANT_ID}'
zenith.zenith_timeline='${TIMELINE}'
zenith.page_server_connstring='host=127.0.0.1 port=6400'" >> zenith_tmp/compute/postgresql.conf

    sh -c "cd zenith_tmp/compute && VCL=`PGDATA=zenith_tmp/compute postgres --sync-walkeepers` && psql -p6400 -hlocalhost -Xqtc 'basebackup ${TENANT_ID} ${TIMELINE} ${VCL}' | tar -xk"
}
```

Reply:
Hm, I wasn't entirely correct with this script. In the cloud control plane currently we do basebackup FIRST and then put config. Anyway, we can handle all scenarios if needed: postgresql.conf exists, empty or not, doesn't exist.

Yet it's still an open question to what extent our 'basebackup' should emulate a real basebackup, I think. Probably we should receive only absolutely necessary data from Pageserver, while all config files and empty directories structure should be created locally by CLI, cloud control plane or whatever frontend is used